backend/reg.ml

```diff
@@ -308,6 +313,7 @@ let equal_location left right =
     false
 
 let same_loc left right =
+  (* CR azewierzejew for xclerc: Shouldn't this also compare register class for Local stack? *)
```

Comment:
It should; the code will not be as straightforward
as one would like because the class is given by
`Proc`, which of course depends on `Reg`...

Reply:
Changed to CR-soon with explanation